.github/workflows/prod-deploy.yml

```diff
@@ -80,4 +80,4 @@ jobs:
           git add values.yaml
           git commit -m "chore: upgrade shopify tracker to $TAG_NAME"
           git push -u origin shopify-tracker-$TAG_NAME
-          hub pull-request -m "chore: upgrade shopify tracker to $TAG_NAME"
```

Comment:
[Nitpick] If title is needed, we can quickly check the cli to understand how to set title for PR

Reply:
`--fill` option in the new command will use the commit as the pr name.